libsolidity/ast/Types.cpp

```diff
@@ -340,11 +364,133 @@ MemberList::MemberMap IntegerType::nativeMembers(ContractDefinition const*) cons
 		return MemberList::MemberMap();
 }
 
-bool IntegerConstantType::isValidLiteral(const Literal& _literal)
+FixedPointType::FixedPointType(int _integerBits, int _fractionalBits, FixedPointType::Modifier _modifier):
+	m_integerBits(_integerBits), m_fractionalBits(_fractionalBits), m_modifier(_modifier)
+{
+	solAssert(
+		m_integerBits + m_fractionalBits > 0 && 
```

Comment:
just simplicity of the idea and especially for explicit conversion. Be odd if we just out of the blue disallowed them from having a fixed act as an int...but I suppose it could go both ways. Me personally I'd like the flexibility in the understanding of it.